R/generate_template.R

```diff
@@ -313,3 +345,53 @@ create_new_application <- function(name, location, sampleapp = FALSE, resetbutto
 
     return()
 }
+
+create_default_theme_file <- function(theme_file) {
+    lines <- c("### primary_color",
+               "# To set primary status color, that affect the color of header, valueBox, infoBox and box",
```

Comment:
Can you rewrite this as "Sets the primary status color" that affects the color of the header, valueBox, infoBox and box. 
Valid values are names of the color or hex-decimal value of the color.

Reply:
done